observability/cluster_observability_operator/cluster-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,43 @@ The {coo-short} complements the built-in monitoring capabilities of {product-tit
 
 These release notes track the development of the {coo-full} in {product-title}.
 
+
+[id="cluster-observability-operator-release-notes-1-1_{context}"]
+== {coo-full} 1.1
+
+// Back fill advisory when it is created by KOnflux
+// The following advisory is available for {coo-full} 1.1:
+//
+// * link:https://access.redhat.com/errata/RHSA-2025:????[RHEA-2025:??? {coo-full} 1.1]
+
+
+[id="cluster-observability-operator-1-1-new-features-enhancements_{context}"]
+=== New features and enhancements
+ 
+* You can now install the monitoring UI plugin using {coo-short}. (link:https://issues.redhat.com/browse/COO-262[*COO-262*])
+
+* You can enable incident detection in the monitoring UI plugin. (link:https://issues.redhat.com/browse/COO-690[*COO-690*])
+
+* TLS support for the Thanos web endpoint has been added. (link:https://issues.redhat.com/browse/COO-222[*COO-222*])
+
+The following table provides information about which features are available depending on the version of {coo-full} and {product-title}:
+
+[cols="1,1,1,1,1,1,1", options="header"]
+|===
+| COO Version   | OCP Versions       | Distributed Tracing   | Logging   | Troubleshooting Panel | Monitoring | Incident detection
+| 1.1           | 4.12 - 4.14        | ✔                     | ✔         | ✘                     | ✘          | ✘
+| 1.1           | 4.15               | ✔                     | ✔         | ✘                     | ✔          | ✘
+| 1.1           | 4.16 - 4.17        | ✔                     | ✔         | ✔                     | ✔          | ✘
+| 1.1           | 4.18+              | ✔                     | ✔         | ✔                     | ✔          | ✔
+|===
```

Comment:
d in `Incident detection` should be capital
`Incident detection` is under monitoring UIplugin, if the last column is 'Incident detection', Monitoring should be changed to 'ACM Alert'

Reply:
I think according to CCS style guidelines, they should all be lower case - will fix